website/public/js/controllers/groupsCtrl.js

```diff
@@ -184,6 +184,12 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
           $scope.$close();
         });
       }
```

Comment:
Oh duh. I forgot that it was all one file. 

Note to self: Make it separate files.